client/extensions/woocommerce/app/dashboard/manage-orders.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { getLink } from '../../lib/nav-utils';
+import ProcessOrdersWidget from '../../components/process-orders-widget';
+import ReadingWidget from '../../components/reading-widget';
+import ShareWidget from '../../components/share-widget';
+import WidgetGroup from '../../components/widget-group';
+
+class ManageOrders extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string.isRequired,
+		} ),
+	};
+
+	renderProcessOrdersWidget = () => {
+		const { site } = this.props;
+		return (
+			<ProcessOrdersWidget className="dashboard__process-orders-widget" site={ site } />
+		);
+	}
+
+	possiblyRenderReviewsWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderInventoryWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderRevenueWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderTopSellersWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderTopReferrersWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderTrafficSpikeWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderSearchHolesWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderExitPagesWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderPoorConvertorsWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderAbandonedCartsWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderTrafficWidget = () => {
+		// TODO - implement in a follow-on PR
+		return null;
+	}
+
+	possiblyRenderShareWidget = () => {
+		// TODO - connect to display preferences in a follow-on PR
+		const { site, translate } = this.props;
+		return (
+			<ShareWidget
+				text={ translate( 'Share a link to your store on social media.' ) }
+				title={ translate( 'Share your store' ) }
+				urlToShare={ getLink( 'https://:site', site ) }
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/5512a8530a3394de9a0cedd0beb566762edddebc